search/geocoder.cpp

```diff
@@ -745,7 +706,7 @@ void Geocoder::FillLocalitiesTable(BaseContext const & ctx)
 
 #if defined(DEBUG)
         ft.GetName(StringUtf8Multilang::kDefaultCode, city.m_defaultName);
-        LOG(LDEBUG, ("City =", city.m_defaultName, radius));
+        LOG(LINFO, ("City =", city.m_defaultName, "radius =", radius, "tfidf =", l.m_tfidf));
```

Comment:
Yes, that's right. But are you sure that you want to have |LINFO| under #ifdef DEBUG?
At the first glance it'll be print in release version. I don't insist. It's just for your information.